cli/tests/unit/tls_test.ts

```diff
@@ -232,3 +233,41 @@ unitTest(
     conn.close();
   },
 );
+unitTest(
+  { perms: { net: true, read: true } },
+  async function getAndConnectWithSNI(): Promise<void> {
+    const resolvable = deferred();
+    const hostname = "localhost";
+    const port = 3600;
+    const certFile = "cli/tests/tls/localhost.crt";
+    const keyFile = "cli/tests/tls/localhost.key";
+    const rootCertFile = "cli/tests/tls/RootCa.pem";
+
+    const listener = Deno.listenTls({
+      hostname,
+      port,
+      certFile,
+      keyFile,
+    });
+
+    listener.accept().then((conn) => {
+      assertEquals((conn as Deno.Conn & { sni?: string }).sni, hostname);
```

Comment:
You need to fix these typings. Please add the correct typings to the `cli/dts/lib.deno.unstable.d.ts` file.

Reply:
I tried adding the typings to `lib.deno.unstable.d.ts`. However, during the tests they were not applied.